### DISCUSSION TITLE
why i cant run code on replit

### DISCUSSION THREAD

**Discussion by tonquoc0407** [mishl-dev/Discord-AI-Chatbot#253] [General]
<img width="485" alt="Screenshot 2024-03-11 112320" src="https://github.com/mishalhossin/Discord-AI-Chatbot/assets/111742025/ac2963ea-c724-46c1-8a6d-8df5636697b3">

**Comment by AmadeusWizard** [comment]
pls read documention, and btw this project works really good on Replit and VPS servers (:
